src/tools/ScenarioMeasurement/Startup/Startup.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
As is, this will allow any runtime that can support netcoreapp2.2 to run this.  Do we care about automatically taking new patch versions if the repo gets upgraded by Arcade?  If so, we should specify the runtimeframeworkversion.  I'm thinking  about this from the angle of controlling variables.

Reply:
I had been planning to just use the runtime under test, because I'm a huge dogfooding fan. It also simplifies our test deployment - right now we only have the runtime under test available to us in the test pipeline. We could of course pull in a 2.2 stable runtime also. 

My guess is that the runtime this runs against is very unlikely to introduce variables that matter (since this process sits mostly quiescent while the test happens), and if we do see those variables I really want to know about them. Let's chat if you think I'm wrong. :)

Reply:
I'll think about this.  Don't block on me.